client/elements/text/sc-image.html

```diff
@@ -0,0 +1,113 @@
+<link rel="import" href="/bower_components/polymer/polymer.html">
+<link rel="import" href="/bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="/bower_components/iron-meta/iron-meta.html">
+
+<link href="/node_modules/viewerjs/dist/viewer.min.css" rel="stylesheet">
+<link rel="import" href="/elements/text/sc-viewer-import.html">
+
+<dom-module id="sc-image">
+  <template>
+
+    <style>
+    </style>
+    
+    <iron-meta id="meta"></iron-meta>
+
+    <iron-ajax
+        id="ajax"
+        debounce-duration="500"
+        handle-as="json"
+        loading="{{isLoading}}"
+        last-error="{{lastError}}"
+        last-response="{{imagesData}}"
+        on-response="_gotData"></iron-ajax>
+
+    <ul id="images" style="display: none;">
+        <template is="dom-repeat" items="[[imagesData]]" as="image">
+            <li><img class="image" data-url="[[_getImageUrl(image.name)]]" alt="[[image.name]]"></li>      
+        </template>
+    </ul>
+
+  </template>
+
+  <script>
+      class SCImage extends Polymer.Element {
+          static get is() {
+              return 'sc-image';
+          }
+
+          static get properties() {
+              return {
+                  division: {
+                      type: String
+                  },
+                  vol: {
+                      type: String
+                  },
+                  page: {
```

Comment:
I think this should be `pageNumber` or `pageNum`. `page` might suggest it's an image.